packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -75,6 +96,14 @@ export default {
 		this.destroyPopper()
 	},
 	methods: {
+		clickHandler(option) {
+			if (
+				!option.isDisabled &&
+				option.onClick &&
```

Comment:
you'll only need this check bc your validator guards against the type of `onClick` if it exists